lib/tori/backend/s3.rb

```diff
@@ -91,6 +91,17 @@ def read(filename)
         body(filename).read
       end
 
+      def public_url(filename)
+        "#{client.config.endpoint}/#{@bucket}/#{filename}"
+      end
+
+      def url_for(filename, method)
+        signer = Aws::S3::Presigner.new(client: client)
+        signer.presigned_url(method, bucket: @bucket, key: filename)
+      end
+
+      private
+
```

Comment:
Why you use their methods even though common methods in backend (such as `read`) exist. We should use common interface, shouldn't we?